taskcluster/xpi_taskgraph/transforms/beetmover.py

```diff
@@ -32,34 +31,37 @@ def add_beetmover_worker_config(config, tasks):
         config.params.get("version")
         and config.params.get("xpi_name")
         and config.params.get("head_ref")
-        and config.params.get("moz_build_date")
+        and config.params.get("build_number")
         and config.params.get("level")
     ):
         manifest = get_manifest()
         xpi_name = config.params["xpi_name"]
         xpi_manifest = manifest[xpi_name]
         xpi_addon_type = xpi_manifest["addon-type"]
+        build_number = config.params["build_number"]
+        xpi_version = config.params["version"]
+        release_name = ("{xpi_name}-{xpi_version}-build{build_number}").format(
+            xpi_name=xpi_name,
+            xpi_version=xpi_version,
+            build_number=build_number,
+        )
         for task in tasks:
-            if xpi_addon_type not in task["only-for-addon-types"]:
-                continue
-            xpi_version = config.params["version"]
             xpi_destinations = []
             for artifact in xpi_manifest["artifacts"]:
-                artifact_name = splitext(basename(artifact))[0]
+                artifact_name = basename(artifact)
                 xpi_destination = (
-                    "pub/system-addons/{xpi_name}/"
-                    "{artifact_name}_mozilla.org-{xpi_version}-{build_id}.xpi"
+                    "pub/system-addons/{xpi_name}/{release_name}/{artifact_name}"
```

Comment:
I think the reason we used splitext was some addons used `.zip` extensions. I'm not sure if there are any anymore.

Reply:
I see. Yea, there's one: [normandy-devtools-1.x](https://github.com/mozilla-extensions/xpi-manifest/blob/main/manifests/normandy-devtools-1.x.yml#L8)